### PR TITLE
Nim 2.0.10 release article

### PR DESCRIPTION
**Pull request by narimiran**

### CODE REVIEW

File: jekyll/_posts/2024-07-18-version-2010-released.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+title: "Nim version 2.0.10 released"
+author: The Nim Team
+---
+
+The Nim team is happy to announce Nim version 2.0.10, our fifth patch release for Nim 2.0.
+
+Version 2.0.10 is a small release, containing just [10 commits](https://github.com/nim-lang/Nim/compare/v2.0.8...v2.0.10),
+but it brings important improvements to Nim 2.0.8, released two weeks ago.
+
+The reason for another quick release is because this version (finally!) fixes the problem with the memory allocator
```
Comment (SirOlaf) [inline jekyll/_posts/2024-07-18-version-2010-released.md:11]: Don’t think this is true, there’s still leaks all over the place.

Comment (Araq) [reply jekyll/_posts/2024-07-18-version-2010-released.md:11]: Bummer. Do you have an example program?

Comment (SirOlaf) [reply jekyll/_posts/2024-07-18-version-2010-released.md:11]: Same one I fixed the segfault on has a small leak if you run in in a loop. This also leaked last time I tested https://github.com/nim-lang/Nim/issues/23361

Comment (narimiran) [reply jekyll/_posts/2024-07-18-version-2010-released.md:11]: @SirOlaf with https://github.com/nim-lang/Nim/pull/23856 and https://github.com/nim-lang/Nim/pull/23871 now backported to 2.0.x, can we now claim this has been fixed?

Comment (SirOlaf) [reply jekyll/_posts/2024-07-18-version-2010-released.md:11]: Yeah it should be, but I do want to mention that there’s still some unsafe behavior when sharing across threads and thread heaps are not freed. That’s why the other example leaks.

**Pull request closed without merging**